Overlay/style.css

```diff
@@ -0,0 +1,12 @@
+@import '../variables.css';
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--outer-space-ultra-dark);
+  overflow-x: hidden;
```

Comment:
Yup, good catch. So, this triggers a scroll effect similar to Trello when we open a card and can scroll it. Reflecting on this approach, that functionality feels like it's best suitable for a component implementation that uses `Overlay` rather than the implementation inheriting this behaviour. 😄 Will get this one removed!